core/src/Cache/CacheFallbackOperations.cs

```diff
@@ -131,7 +140,10 @@ internal class CacheFallbackOperations
             }
             catch (Exception ex)
             {
-                CoreLoggerBase.Default.Warning("GetAllAdalUsersForMsal failed due to Exception - " + ex);
+                string msg = "GetAllAdalUsersForMsal failed due to Exception - ";
```

Comment:
>GetAllAdalUsersForMsal failed due to Exception -  [](start = 30, length = 49)

in general we should not use messages which are tied to our code, rather the message should say something that makes sense for a developer (example provided above as well). For this one it could be: 
An error occurred while searching and matching adal users in the msal cache.

Reply:
done.